src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.subsystems.gripper;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+
+public class Gripper extends LoggedSubsystem<GripperLoggedInputs> {
+    private static Gripper INSTANCE;
+
+    private final Solenoid solenoid = new Solenoid(PneumaticsModuleType.CTREPCM, Ports.Gripper.SOLENOID);
+
+    /**
+     * @return the instance of the subsystem.
+     */
+    public static Gripper getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new Gripper();
+        }
+        return INSTANCE;
+    }
+
+    private Gripper() {
+        super(new GripperLoggedInputs());
+    }
+
+    /**
+     * Open the Gripper.
+     */
+    public void open() {
+        solenoid.set(true);
+    }
+
+    /**
+     * Close the Gripper.
+     */
+    public void close() {
+        solenoid.set(false);
+    }
+
+    /**
+     * Changes the current state to the other state.
+     */
+    public void toggle() {
+        solenoid.toggle();
+    }
+
+    /**
+     * @return the current state of the solenoids.
```

Comment:
1. There is only one solenoid.
2. End-users don't need to know the gripper has a solenoid.